quickstart/aspnet-core/ContosoTeamStats/RedisConnection.cs

```diff
@@ -69,6 +69,7 @@ public async Task<T> BasicRetryAsync<T>(Func<IDatabase, Task<T>> func)
                     }
                     catch (ObjectDisposedException) { }
                 }
+                catch (ObjectDisposedException) {}
```

Comment:
This means if the `func()` throws `ObjectDisposedException`, then it fails silently without force reconnecting. I think instead we want to add `ObjectDisposedException` to the list of exception types we catch on line 58

I suspect we should also be retrying `func()` in the catch block after calling `ForceReconnectAsync()` (and return the result)